### DISCUSSION TITLE
Discussion topics for Apr 4, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4999] [Show and tell]
Discussion topics for Apr 4, 2022 Engineering Sync

**Comment by sethdarragile6** [comment]
Quick review of [PR ](https://github.com/CDCgov/prime-reportstream/pull/4955) for [2091](https://app.zenhub.com/workspaces/experience-607d9d5e68b95200150fec37/issues/cdcgov/prime-reportstream/2091)
1. Generally, can anyone see any more potential downstream issues with this, and
2. Specifically, is there special handling around the async processing use case (processRequest() line 219 ReportFunction.kt

**Comment by jimduff-usds** [comment]
3.  How to use the History API to do debugging
